src/infra/main/template.yaml

```diff
@@ -134,23 +148,46 @@ Resources:
         - Key: Source
           Value: !Ref SourceTagValue
 
-  AISSampleLambdaHandlerPolicy:
+  InterventionProcessorFunctionPolicy:
     Type: AWS::IAM::Policy
     Properties:
-      PolicyName: !Sub "${AWS::StackName}-AISSampleLambdaHandlerPolicy"
+      PolicyName: !Sub "${AWS::StackName}-InterventionProcessorFunctionPolicy"
       PolicyDocument:
         Version: "2012-10-17"
         Statement:
           - Effect: Allow
             Action:
-              - "dynamodb:PutItem"
-              - "dynamodb:Query"
-              - "dynamodb:DeleteItem"
+              - "logs:CreateLogStream"
+              - "logs:PutLogEvents"
+            Resource:
+              - !GetAtt InterventionProcessorFunctionRole.Arn
```

Comment:
I believe this should be the log group ARN

Reply:
removed the whole statement as we are not creating the log group explicitly - we can add it later on if needed